public/main.css

```diff
@@ -356,6 +357,22 @@ h2 {
   filter: grayscale(0%);
 }
 
+#description {
+  display: none;
+}
+
+#description:checked + label {
+  height: 100%;
+}
+.description-text {
```

Comment:
this should probably have a hover cursor change, to hint at the fact that it's interactable